hdtv/database/DDEPLibraries.py

```diff
@@ -18,25 +19,29 @@
     Opens table of nuclides with peak energies, gives back the peak energies of the nuclide and its intensities.
     """
 
-    out = {'nuclide': nuclide, 'transitions': []}
+    out = {"nuclide": nuclide, "transitions": []}
 
     if nuclide == "Ra-226":
         nuclide = "Ra-226D"
 
     try:
-        with urllib.request.urlopen("http://www.nucleide.org/DDEP_WG/Nuclides/" + str(nuclide) + ".lara.txt") as resource:
+        with urllib.request.urlopen(
```

Comment:
Codacy found an issue: [Audit url open for permitted schemes. Allowing use of file:/ or custom schemes is often unexpected.](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)